### DISCUSSION TITLE
앱 알림 기획

### DISCUSSION THREAD

**Discussion by 14KGun** [sparcs-kaist/taxi-app#15] [Ideas]
## 앱 알림 기획
 앱 알림 조건들에 대해서 정리합니다.
 - 택시 출발 10분 전 알림
   - 마이페이지에서 on/off 가능
 - 채팅 알림 (참여 중인 방에서 채팅 메시지를 받았을 경우, 다만 방에 참여 중인 경우 알람을 받으면 안되겠죠?)
   - 마이페이지에서 on/off 가능
 
@imYourChoi @happycastle114

**Comment by happycastle114** [comment]
- 채팅 알림 고려사항
앱이나 웹에서 채팅창을 켜두고 있는 경우에는 알림을 보내면 안됨.
앱이 백그라운드에서 켜져 있는 경우에는 알림을 보내야 함.

**Comment by happycastle114** [comment]
- 키워드 알림
키워드(경로)를 미리 등록해두고 누군가 해당 경로에 맞는 게시물을 등록하면 알림을 보냄.
